project-base/docker/php-fpm/Dockerfile

```diff
@@ -1,58 +1,51 @@
-FROM phpdockerio/php72-fpm:latest
-
-# Switch to non-interactive mode during image build to silence some warnings.
-# See https://github.com/moby/moby/issues/4032
-ARG DEBIAN_FRONTEND=noninteractive
-
-# Install packages are useful when working in the container in interactive mode.
-RUN apt-get update && apt-get -y install apt-utils dialog
-
-# Install ACL support (ie. tools like "setfacl").
-RUN apt-get update && apt-get -y install acl
-
-# Install Git - it is necessary for code diffs
-RUN apt-get update && apt-get -y install git
+FROM php:7.2-fpm-alpine
+
+# install git for computing diffs
+RUN apk add --update git
+
+# install Composer
+COPY docker-install-composer /usr/local/bin/docker-install-composer
+RUN chmod +x /usr/local/bin/docker-install-composer && docker-install-composer
+
+# libpng-dev needed by "gd" extension
+# icu-dev needed by "intl" extension
+# postgresql-dev needed by "pgsql" extension
+# libzip-dev needed by "zip" extension
+# autoconf needed by "redis" extension
+RUN apk add --update \
+    libpng-dev \
+    icu-dev \
+    postgresql-dev \
+    libzip-dev \
+    autoconf
+
+# "zip" extension warns about deprecation if we do not use a system library
+RUN docker-php-ext-configure zip --with-libzip
+
+# install necessary PHP extensions requested by Composer
+RUN docker-php-ext-install \
```

Comment:
In the previous version we had specified 16 PHP extension, now we have only 8, how so?

Reply:
They were installed by default.